spring-cloud-starter-dataflow-server-local/src/test/java/org/springframework/cloud/dataflow/server/single/security/LocalServerSecurityWithOAuth2Tests.java

```diff
@@ -59,6 +63,16 @@ public void testAccessRootUrlWithBasicAuthCredentials() throws Exception {
 				.andExpect(status().isOk());
 	}
 
+	@Test
+	public void testAccessRootUrlWithBasicAuthCredentialsTwice() throws Exception {
+		localDataflowResource.getMockMvc()
+				.perform(get("/").header("Authorization", basicAuthorizationHeader("user", "secret10"))).andDo(print())
+				.andExpect(status().isOk());
+		localDataflowResource.getMockMvc()
```

Comment:
I guess we should eventually have some assert that the token is cached when making the second call.